build/RELEASING.md

```diff
@@ -44,49 +39,57 @@ Only for Maintainers.
 ```
 
    This generates combined changelog to be used in Github release.
-   Once contents of combined changelog is saved to Github release,
+   Once contents of combined changelog is saved somewhere,
    delete the file.
 
 3.Run the following PowerShell script from the root of the repo.
    This updates all the changelog to have release date for the
    current version being released.
    Replace the version with actual version.
-   The actual version would be the tag name from step1 appended with
-   ".1"
+   The actual version would be the tag name from step1.
 
 ```powershell
      $changelogs = Get-ChildItem -Path . -Recurse -Filter changelog.md
     foreach ($changelog in $changelogs)
     {
      (Get-Content -Path $changelog.FullName) -replace "Unreleased", "Unreleased
 
-## 0.7.0-beta.1
+## 1.0.0-rc2
 
 Released $(Get-Date -UFormat '%Y-%b-%d')" | Set-Content -Path $changelog.FullName
     }
 ```
 
 4.Submit PR with the above changes, and get it merged.
 
-5.Open [Pack and publish to MyGet
+5. Tag Git with version to be released e.g.:
+
+   ```sh
+   git tag -a 1.0.0-rc2 -m "1.0.0-RC2"
+   git push origin 1.0.0-rc2
+   ```
+
+We use [MinVer](https://github.com/adamralph/minver) to do versioning,
+which produces version numbers based on git tags.
+
+6.Open [Pack and publish to MyGet
```

Comment:
The key change is, we tag and then run release workflow. So the package will be named exactly as the tag name. i.e if tag was 1.0.0, version would be 1.0.0.
Then the daily myget publish will automatically do, 1.0.1-alpha.1 series.
When we release 1.0.1, we tag 1.0.1 and run release workflow which push 1.0.1 nugets.
Then daily myget publish starts 1.0.2-alpha.1 series.
(all of which is easily customizable by pushing required tags)

Reply:
The PR title has a typo which needs to be fixed.